src/components/Notification/NotificationStack.tsx

```diff
@@ -0,0 +1,55 @@
+import { createPortal } from 'react-dom'
+import styled from 'styled-components'
+
+import { useNotificationStore } from '@/stores/notification'
+
+import { createContainer } from './container'
+import { Notification } from './Notification'
+
+const NotificationStackContainer = styled.div`
+  position: relative;
+  width: 200px;
+  > div:nth-child(3) {
+    top: 28px;
+    z-index: 3;
```

Comment:
If I have a element with `position: absolute` and `z-index: 4`, does it overlap with this element?

Reply:
> the NotificationStack is put in the Container with z-index: 9999, so every component has z-index fewer than 9999 will be overlaid by the Notification. Lemme know if this way is ok

cool

Reply:
Does NotificationStack always overlap with other elements? If it does, I suggest you use `2147483647` for `z-index`. 

https://stackoverflow.com/questions/491052/minimum-and-maximum-value-of-z-index